homeassistant/components/jellyfin/diagnostics.py

```diff
@@ -0,0 +1,48 @@
+"""Diagnostics support for Jellyfin."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .models import JellyfinData
+
+TO_REDACT = {CONF_PASSWORD}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    data: JellyfinData = hass.data[DOMAIN][entry.entry_id]
+    sessions = data.coordinators["sessions"]
+
+    return {
+        "entry": {
+            "title": entry.title,
+            "data": async_redact_data(entry.data, TO_REDACT),
+        },
+        "server": {
+            "id": sessions.server_id,
+            "name": sessions.server_name,
+            "version": sessions.server_version,
+        },
+        "sessions": [
+            {
+                "id": session_id,
+                "user_id": session_data.get("UserId"),
```

Comment:
Is there any chance there is public personal information tied to this UUID?

Reply:
I wouldn't expect so atleast in the ways I've used jellyfin. It's a local account tied to server. There maybe we ways to use like LDAP or other authors mechanism that could log it and perhaps the streaming devices could store it on login. It's geared for local private usage primarily. Remote access even takes a bit to setup.

Reply:
from the website: "Jellyfin has no tracking, phone-home, or central servers collecting your data. We believe in keeping our software open and transparent. We’re also not in the media business, so the only media you see is your own."